docs/contributor/03-11-hap-rules.md

```diff
@@ -0,0 +1,293 @@
+
+| Plan                	| Hyperscaler Type 	|
+|---------------------	|------------------	|
+| azure               	| azure            	|
+| azure_lite          	| azure            	|
+| aws                 	| aws              	|
+| gcp                 	| gcp              	|
+| preview             	| aws              	|
+| sap-converged-cloud 	| openstack_<HR>   	|
+| trial               	| aws              	|
+
+Rest of the examples in the document follow the same structure of presenting an example configuration and corresponding HAP pools as above snippet.
+
+## Rule Attributes
+
+Triggered rule entry applies that labels to the SecretBinding search query.
+Rule entry attributes and its plan alter the search query by adding new or modifing existing [search labels](#search-labels). 
+Existence of an attribute in the rule means its inclusion in the [search labels](#search-labels) in addition to labels associated with rule plan.
+
+### Platform Region Attribute
+
+It is possible to extend the rule entry with specified platform region by adding  a `PR` rule attribute. The value of PR attributes is compared to region of a subaccount of SKR and, if triggered, applies `hyperscalerType: <HYPERSCALER_NAME>[_<PLATFORM_REGION>]` label.
+
+Below configuration means that if a gcp cluster is provisioned in cf-sa30 subaccount region then KEB would search for secret bindings with `hyperscalerType: gcp_cf-sa30` label. 
+
+```
+hap: 
+  rule: 
+    - gcp(PR=cf-sa30)
+
+SecretBinding pools
+- hyperscalerType: gcp_cf-sa30, 
+```
+
+### Hyperscaler Region Attribute
+
+A region that an SKR is provisioned in can be matched with an attribute named `HR`. Below configuration specifies azure plan as one for which to use hyperscaler region based search. In this case, cluster provisioned in with the azure plan would require a secret binding with `hyperscalerType: azure_<HYPERSCALER_REGION>` label. 
+
+```
+hap: 
+  rule: 
+    - gcp(HR=us-central1)
+
+SecretBinding pools
+- hyperscalerType: gcp_us-central1, 
+```
+
+### Shared & EU Access attribute
+
+The `shared` and `euAccess` attributes does not correspond to any physical SKR property. It is a logical attribute that is used only to add its label. If the rule entry contains `shared` or `euAccess` attributes then when triggered a `shared: true` or `euAccess: true` labels are added to the [search labels](#search-labels) depending on which one of those two attributes is listed in the rule entry. Shared label on a SecretBindings marked it as assignable to more than one SKR and euAccess is a non-functional label not reused by KEB (more on that can be found in p[HAP Pool document](03-10-hyperscaler-account-pool.md)). Below configuration specifies that all gcp clusters will use the same pool of shared secret bindings marked with labels `hyperscalerType: gcp`, `shared: true` and azure clusters in the region cf-ch20 will use the pool of secret bindings marked with labels `hyperscalerType: azure`, `euAccess: true`. 
```

Comment:
Still I am not convinced that we should use terms logical and physical attribute. The only physical is HR.
So I suggest to keep it simple and not introduce this distinction.

Reply:
What does it mean:
> not reused by KEB

Reply:
And here we have `logical` leftover.

Reply:
Corrected.